data/matrix_layouts/razerdefault22.json

```diff
@@ -267,5 +267,141 @@
+            {"label": "9", "matrix": [1, 10]},
+            {"label": "0", "matrix": [1, 11]},
+            {"label": "+", "matrix": [1, 12]},
+            {"label": "\u00b4", "matrix": [1, 13]},
+            {"label": "backspace", "width": 126, "matrix": [1, 14]},
+            {"label": "ins", "matrix": [1, 15]},
+            {"label": "home", "matrix": [1, 16]},
+            {"label": "page\nup", "matrix": [1, 17]},
+            {"label": "num\nlk", "matrix": [1, 18]},
+            {"label": "/", "matrix": [1, 19]},
+            {"label": "*", "matrix": [1, 20]},
+            {"label": "-", "matrix": [1, 21]}
+        ],
+        "row2": [
+            {"label": "M2", "matrix": [2, 0]},
+            {"label": "\u21e9", "width": 94, "matrix": [2, 1]},
+            {"label": "Q", "matrix": [2, 2]},
+            {"label": "W", "matrix": [2, 3]},
+            {"label": "E", "matrix": [2, 4]},
+            {"label": "R", "matrix": [2, 5]},
+            {"label": "T", "matrix": [2, 6]},
+            {"label": "Y", "matrix": [2, 7]},
+            {"label": "U", "matrix": [2, 8]},
+            {"label": "I", "matrix": [2, 9]},
+            {"label": "O", "matrix": [2, 10]},
+            {"label": "P", "matrix": [2, 11]},
+            {"label": "\u00c5", "matrix": [2, 12]},
+            {"label": "\u0308", "matrix": [2, 13]},
+            {"label": "enter", "width": 92, "matrix": [3, 14]},
+            {"label": "del", "matrix": [2, 15]},
+            {"label": "end", "matrix": [2, 16]},
+            {"label": "page\ndown", "matrix": [2, 17]},
+            {"label": "7", "matrix": [2, 18]},
+            {"label": "8", "matrix": [2, 19]},
+            {"label": "9", "matrix": [2, 20]},
+            {"label": "+", "height": 126, "matrix": [2, 21]}
+        ],
+        "row3": [
+            {"label": "M3", "matrix": [3, 0]},
+            {"label": "caps", "width": 109, "matrix": [3, 1]},
+            {"label": "A", "matrix": [3, 2]},
+            {"label": "S", "matrix": [3, 3]},
+            {"label": "D", "matrix": [3, 4]},
+            {"label": "F", "matrix": [3, 5]},
+            {"label": "G", "matrix": [3, 6]},
+            {"label": "H", "matrix": [3, 7]},
+            {"label": "J", "matrix": [3, 8]},
+            {"label": "K", "matrix": [3, 9]},
+            {"label": "L", "matrix": [3, 10]},
+            {"label": "\u00d6", "matrix": [3, 11]},
+            {"label": "\u00c4", "matrix": [3, 12]},
+            {"label": "'", "matrix": [3, 13]},
+            {"label": "enter", "width": 77, "disabled": true},
+            {"label": null},
+            {"label": null},
+            {"label": null},
+            {"label": "4", "matrix": [3, 18]},
+            {"label": "5", "matrix": [3, 19]},
+            {"label": "6", "matrix": [3, 20]},
+            {"label": "+", "disabled": true}
+        ],
+        "row4": [
+            {"label": "M4", "matrix": [4, 0]},
+            {"label": "\u21e7", "width": 80, "matrix": [4, 1]},
```

Comment:
`shift`?

Reply:
That's probably more correct, yeah. I just copied the shift-key glyph that was used by other keyboards, to keep it consistent in that manner, not consistent with the physical print on the keyboard.